bayesian_cnn_prometheus/analysis/masks_analyzer.py

```diff
@@ -0,0 +1,75 @@
+import json
+import os
+import glob
+
+from bayesian_cnn_prometheus.analysis.similarity_comparer import SimilarityComparer
+# refactor idea: importing get_patient_index from there; maybe it should be in another place?
```

Comment:
To jest bardzo słuszny refactor idea ^^ Może by utworzyć plik utils.py w folderze bayesian_cnn_prometheus i tam przenieść get_patient_index i load_nifti_file i podmienić te funkcje w DataSplitter i w ewaluacji? Jeśli dasz radę to mozesz to zrobić, a jeśli nie to zrobimy to w innym PR :)